xml/System.Globalization/StringInfo.xml

```diff
@@ -491,9 +491,9 @@ This example uses the <xref:System.Globalization.StringInfo.GetTextElementEnumer
         <Parameter Name="str" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="str">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="str">The input span to analyze.</param>
+        <summary>Returns the length of the first text element (extended grapheme cluster) that occurs in the input span.</summary>
```

Comment:
>(extended grapheme cluster) [](http://example.com/codeflow?start=62&length=27)

this part can be confusing as the first element in the text could be not a grapheme cluster and can bee a normal single character.

Maybe remove this part and add extra sentence telling  the first element could be grapheme cluster or surrogate pair which can have length greater than one character.